tests/Gedmo/Timestampable/Fixture/WithoutInterface.php

```diff
@@ -8,26 +8,39 @@
 /**
  * @ORM\Entity
  */
+#[ORM\Entity]
 class WithoutInterface
 {
-    /** @ORM\Id @ORM\GeneratedValue @ORM\Column(type="integer") */
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
```

Comment:
Can you please also use `Doctrine\DBAL\Types\Types` constants in this file?

Reply:
I thought I already had, but it seems I missed this one...